src/set_theory/game/basic.lean

```diff
@@ -526,7 +526,20 @@ inductive inv_ty (l r : Type u) : bool → Type u
 | right₁ : l → inv_ty ff → inv_ty tt
 | right₂ : r → inv_ty tt → inv_ty tt
 
-instance (l r : Type u) : inhabited (inv_ty l r ff) := ⟨inv_ty.zero⟩
+instance (l r : Type u) [is_empty l] [is_empty r] : is_empty (inv_ty l r tt) :=
+⟨by { rintro (a | a), all_goals { exact is_empty_elim ᾰ_ᾰ } }⟩
```

Comment:
I couldn't get the variable `a` to appear for whatever reason? Help wanted.

Reply:
I'm not sure what's going on with `rintro` exactly, but here are some tricks to be able to avoid using `ᾰ_ᾰ` by name: 
```lean
instance (l r : Type u) [is_empty l] [is_empty r] : is_empty (inv_ty l r tt) :=
⟨by { rw [is_empty_iff] at *, rintro ⟨_⟩; solve_by_elim }⟩
```

Reply:
I think I'll stick to the new proof I gave
```lean
instance (l r : Type u) [is_empty l] [is_empty r] : is_empty (inv_ty l r tt) :=
⟨λ h, by { cases h with _ _ _ _ a _ a _; exact is_empty_elim a }⟩
```
It's a bit uglier, but it avoids `solve_by_elim`.

Reply:
I'm not sure there's a reason to avoid `solve_by_elim` for this, but that's ok.